charts/astronomer/templates/houston/worker/houston-worker-pod-disruption-budget.yaml

```
@@ -0,0 +1,18 @@
+###################################################
+## Astronomer Houston Worker Pod Disruption Budget
+###################################################
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Release.Name }}-houston-worker-pdb
+spec:
+  {{- if lt ( .Values.houston.worker.replicas | int ) 4 }}
+  maxUnavailable: 1
+  {{- else }}
+  maxUnavailable: {{ .Values.houston.worker.replicas.maxUnavailable }}
+  {{- end }}
```

```suggestion
  {{- if lt ( .Values.houston.worker.replicas | int ) 2 }}
  minAvailable: 0
  {{- else if lt ( .Values.houston.worker.replicas | int ) 4 }}
  minAvailable: 1
  {{- else }}
  minAvailable: "25%"
  {{- end }}
```
I think that Values.houston.worker.replicas.maxUnavailable is not an existing configuration, so this doesn't seem quite right. But also, do we need this availability level for this component if the expected behavior is dequeuing from nats? i.e. if the pods are down that doesn't interrupt service because they come back up, then continue dequeuing.

---

ah..that's true.  They will be able to replay from NATS.  

I was mainly trying to match this with what we were doing for Houston-API.  I think that the workers should be highly available because eventually deployments won't go out if there aren't any workers up to receive the messages and run the commander calls to deploy.